### DISCUSSION TITLE
Blender AgX and Filmic

### DISCUSSION THREAD

**Discussion by r41dy** [sobotka/AgX#18] [Q&A]
Is there a way to install AgX but leave Filmic so i can use both?

**Comment by danielraziel** [comment]
just install blender 4.0 alpha
or wait a week for beta version

**Comment by sobotka** [comment]
There are different variations, and I was not involved in the authorship of the scripts of the Blender configuration that bears the “AgX” moniker. Broadly, concerning my noodling:
1. “AgX” has an “inset” of primaries, then a curve, and then treats that as BT.709 “as is”.
2. “SB2383” uses a second matrix after the curve and clip.
3. “Resolve” uses a secondary matrix after the curve and clip.

I am unsure whether there is the mandatory explicit clip in the Blender version, and instead I believe it leans on a different mechanic.
